api-specs.json

```diff
@@ -21,7 +21,7 @@
     "dir": "azure-mgmt-keyvault",
     "source": "specification/keyvault/resource-manager/readme.md",
     "package": "com.azure.management.keyvault",
-    "args": "--payload-flattening-threshold=1 --tag=package-2018-02"
+    "args": "--payload-flattening-threshold=1 --tag=package-2018-02 --track2-naming=true"
```

Comment:
For track 2(vnext) branch, this should be the default option.

Reply:
OK. I will change the option to track1-naming.